yarn-project/archiver/src/archiver/archiver.ts

```diff
@@ -84,8 +88,23 @@ export class Archiver implements ArchiveSource {
     private readonly registryAddress: EthAddress,
     private readonly store: ArchiverDataStore,
     private readonly pollingIntervalMs = 10_000,
+    telemetry: TelemetryClient,
     private readonly log: DebugLogger = createDebugLogger('aztec:archiver'),
-  ) {}
+  ) {
+    const meter = telemetry.getMeter('Archiver');
+    this.blockHeight = meter.createGauge(Metrics.ARCHIVER_BLOCK_HEIGHT, {
```

Comment:
If the meter is `Archiver`, is it redundant to call the gauge "ARCHIVER_"? Should/Could it be the case that components other than the Archiver want to use a more generic `Metrics.BLOCK_HEIGHT`?